Confus/Common/TeamScore.cpp

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+#include <memory>
+#include <RakNet/BitStream.h>
+#include <RakNet\MessageIdentifiers.h>
+#include <string>
+
+#include "TeamScore.h"
+#define DEBUG_CONSOLE
+#include "Debug.h"
+
+namespace ConfusServer 
+{
+    void TeamScore::setConnection(Networking::Connection* a_Connection)
+    {
+        m_Connection = a_Connection;
+    }
+
+    void TeamScore::sendScoreToClients()
+    {
+        RakNet::BitStream bitStream;
+        bitStream.Write(static_cast<RakNet::MessageID>(Networking::Connection::EPacketType::ScoreUpdate));
+
+        bitStream.Write(m_RedTeamScore);
+		bitStream.Write(m_BlueTeamScore);
+
+        m_Connection->sendPacketToAllClients(bitStream);
+    }
+
+    void TeamScore::sendWin()
+    {
+        //TODO: Send win to clients
```

Comment:
I see that we don't have a card yet that covers this TODO, and it isn't part of the definition of done of Score.